### DISCUSSION TITLE
CMD Prompt Randomly will close on server after scheduling checkin

### DISCUSSION THREAD

**Discussion by edshaw42** [jdholtz/auto-southwest-check-in#173] [Q&A]
Hello,

I have this running on a server at my house. Have a .bat file created that I just kick off that is attached below. It log's in and works perfect! The only issue I run into is that I have to kick that .bat file off a couple hours before the flight or I will come back and CMD prompt has just randomly closed? It's odd and I have never witnessed it happening. It's just like after a few hours it will just close the window. 

Is there something I am missing here to keep this from happening? Thought about kicking the .bat file off in the job scheduler like 20 mins before my flight. 

`@echo on

cd /auto-southwest-check-in

py southwest.py`

**Comment by jdholtz** [comment]
I don't have experience with `.bat` files, so I'm not exactly sure on their behavior. Are there logs of the script throwing an exception in the `logs/` directory?
